config/zones.json

```diff
@@ -4151,63 +4151,114 @@
       "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts",
       "productionPerUnit": "ENTSOE.fetch_production_per_units"
     },
+    "subZoneNames": ["SE-SE1", "SE-SE2", "SE-SE3", "SE-SE4"],
     "timezone": "Europe/Stockholm"
   },
   "SE-SE1": {
     "bounding_box": [
       [14.926272420000089, 63.65992801849613],
       [24.663413534000114, 69.53635569300009]
     ],
+    "capacity": {
+      "biomass": 271,
+      "coal": 0,
+      "gas": 1,
+      "geothermal": 0,
+      "hydro": 5271,
+      "nuclear": 0,
+      "solar": 19,
+      "wind": 1927
+    },
     "parsers": {
       "consumption": "ENTSOE.fetch_consumption",
       "consumptionForecast": "ENTSOE.fetch_consumption_forecast",
       "generationForecast": "ENTSOE.fetch_generation_forecast",
       "price": "ENTSOE.fetch_price",
-      "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts"
+      "production": "ENTSOE.fetch_production",
+      "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts",
+      "productionPerUnit": "ENTSOE.fetch_production_per_units"
     },
-    "timezone": null
+    "timezone": "Europe/Stockholm"
   },
   "SE-SE2": {
     "bounding_box": [
       [11.492425171000093, 60.280991513822045],
       [21.486074487958014, 66.78799631800014]
     ],
+    "capacity": {
+      "biomass": 680,
+      "coal": 0,
+      "gas": 2,
+      "geothermal": 0,
+      "hydro": 8077,
+      "nuclear": 0,
+      "solar": 94,
+      "wind": 5342
+    },
     "parsers": {
       "consumption": "ENTSOE.fetch_consumption",
       "consumptionForecast": "ENTSOE.fetch_consumption_forecast",
       "generationForecast": "ENTSOE.fetch_generation_forecast",
       "price": "ENTSOE.fetch_price",
-      "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts"
+      "production": "ENTSOE.fetch_production",
+      "productionPerModeForecast": "ENTSOE.fetch_wind_solar_forecasts",
+      "productionPerUnit": "ENTSOE.fetch_production_per_units"
     },
-    "timezone": null
+    "timezone": "Europe/Stockholm"
   },
   "SE-SE3": {
     "bounding_box": [
       [10.608164910000113, 56.392035223000065],
       [19.8385522800001, 62.67478628775066]
     ],
+    "capacity": {
+      "biomass": 2561,
+      "coal": 0,
+      "gas": 950,
+      "geothermal": 0,
```

Comment:
My quick comparison of country wide vs zones capacities shows some discrepencies:
* As noted above, gas is much higher for subzones (1535 vs 698)
* oil is reported at 2251MW for the country while subzones sum up to 905.

For the rest it looks OKish

Reply:
I know, I switched to the official report by SVK (the Swedish TSO) and they appears to be reporting gas together with small scale diesel and gasoline generators, I plan to update this as soon as I find a source that splits it or when I have the time to calculate it myself based on known power plants.

Reply:
Alright, we can make a follow up adjustment. Considering the small share of oil / gas in the Swedish mix it's not crucial we get this 100% right from the get go